src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -224,6 +240,21 @@ class Breakpoints extends PureComponent {
       click: () => toggleBreakpoints(true, otherEnabledBreakpoints)
     };
 
+    const removeCondition = {
+      id: "node-menu-remove-condition",
+      label: removeConditionLabel,
+      accesskey: removeConditionKey,
+      disabled: false,
+      click: () => setBreakpointCondition(breakpoint.location, "")
```

Comment:
Is there any need for the second parameter? i.e should it be just `click: () => setBreakpointCondition(breakpoint.location)`

Reply:
Will change:)